tests/utils/validator_manager.go

```diff
@@ -929,14 +929,34 @@ func InitializeEndNativeValidation(
 	return WaitForTransactionSuccess(ctx, subnet, tx.Hash())
 }
 
+func ForceInitializeEndNativeValidation(
+	ctx context.Context,
+	sendingKey *ecdsa.PrivateKey,
+	subnet interfaces.SubnetTestInfo,
+	stakingManager *nativetokenstakingmanager.NativeTokenStakingManager,
+	validationID ids.ID,
+) *types.Receipt {
+	opts, err := bind.NewKeyedTransactorWithChainID(sendingKey, subnet.EVMChainID)
+	Expect(err).Should(BeNil())
+	tx, err := stakingManager.ForceInitializeEndValidation(
+		opts,
+		validationID,
+		false,
+		0,
+	)
+	Expect(err).Should(BeNil())
+	return WaitForTransactionSuccess(ctx, subnet, tx.Hash())
+}
+
 func InitializeEndERC20Validation(
 	ctx context.Context,
-	senderKey *ecdsa.PrivateKey,
+	sendingKey *ecdsa.PrivateKey,
```

Comment:
Can we change these back to `senderKey` for the standardization. This change as made in https://github.com/ava-labs/teleporter/pull/576